services/server/src/server/services/utils/database-util.ts

```diff
@@ -93,25 +93,42 @@ export namespace Tables {
 
 export interface DatabaseColumns {
   bytecodeHashes: {
-    recompiledCreation?: Hash;
-    recompiledRuntime: Hash;
-    onchainCreation?: Hash;
-    onchainRuntime: Hash;
+    keccak: {
+      recompiledCreation?: BytesKeccak;
+      recompiledRuntime: BytesKeccak;
+      onchainCreation?: BytesKeccak;
+      onchainRuntime: BytesKeccak;
+    };
   };
-  compiledContract: Partial<Tables.CompiledContract>;
-  verifiedContract: Partial<Tables.VerifiedContract>;
+  compiledContract: Pick<
+    Tables.CompiledContract,
+    | "language"
+    | "fully_qualified_name"
+    | "compilation_artifacts"
+    | "creation_code_artifacts"
+    | "runtime_code_artifacts"
+  >;
```

Comment:
Why are you explicitly picking fields here and not the full Tables.CompiledContract type?

Reply:
Because we need all fields but not id

Reply:
Because we need all fields but not id

Reply:
the alternative is 

```
  verifiedContract: Omit<
    Tables.VerifiedContract,
    "id" | "compilation_id" | "deployment_id"
  >;
```

Reply:
Wait do you mean

```ts
  compiledContract: Omit<  
    Tables.CompiledContract,  
    "id"
  >;  
```

I also think this is much more concise and understandable